packages/block-library/src/template-part/edit/label.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * WordPress dependencies
+ */
+import { useEntityProp } from '@wordpress/core-data';
+import { Icon } from '@wordpress/components';
+
+export default function TemplatePartLabel( { postId, slug } ) {
+	const [ title ] = useEntityProp(
+		'postType',
+		'wp_template_part',
+		'title',
+		postId
+	);
```

Comment:
I don't have a strong opinion on this, but figured I would point it out.  It seems we are also using this `useEntityProp` for title in the name-panel component.  Maybe it would make sense to move it into index and pass it down to these two child components?  But maybe it doesn't really matter and is more simple this way.

Reply:
I don't have a strong opinion either. If this becomes widely shared, I'd err on the side of removing implicit dependencies and moving `useEntityProp` out of the label component. The consumer can always pass the value through props.